README.md

```diff
@@ -216,6 +216,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Mini Golf](http://drawvr.com/minigolf/) (by [@drawvr]) - 9 Holes
 - [Welcome to A-Frame](http://drawvr.com/welcome/) (by [@drawvr]) - Quick introduction to A-Frame
 - [Photo Sphere Diver](https://www.ideaspacevr.org/themes/web-vr-photo-sphere-diver) (by [IdeaSpace](https://www.ideaspacevr.org/)) - Dive into your photo spheres
+- [Sketch VR](http://uniform-futures.github.io/sketch-vr/) (by [@uniformfutures]) -  See your paper doodle in VR
```

Comment:
`http://uniform-futures.github.io/sketch-vr/` could be `https://uniform-futures.github.io/sketch-vr/`